frontend/app/src/locales/en.json

```diff
@@ -3100,7 +3100,8 @@
   },
   "decentralized_overview": {
     "loading": "Please wait while your defi balances are getting loaded...",
-    "title": "Defi Overview"
+    "title": "Defi Overview",
+    "empty_title": "No DeFi overview can be shown"
```

Comment:
When can that happen @lukicenturi ?

Reply:
Ah I see. Can you please add a tiny bit more info to the message so the users knows why?

"No DeFi overview can be shown, please add at least one ethereum account" ?

Reply:
> Ah I see. Can you please add a tiny bit more info to the message so the users knows why?
> 
> "No DeFi overview can be shown, please add at least one ethereum account" ?

Updated 
<img width="1236" alt="image" src="https://user-images.githubusercontent.com/26648140/157301113-719e4439-cc49-4d42-a40d-039a671dde4a.png">